bookserver/schemas.py

```diff
@@ -17,6 +17,7 @@
 # Third-party imports
 # -------------------
 from pydantic import BaseModel, BaseConfig, create_model, constr, validator, Field
+from humps import camelize  # type: ignore
```

Comment:
Why the `# type: ignore` here?

Reply:
Because mypy was being a pain and claiming it was not used.